components/cam/src/control/physconst.F90

```diff
@@ -74,12 +74,12 @@ module physconst
    real(r8), public           :: mwdry        = shr_const_mwdair! molecular weight dry air
    real(r8), public           :: rearth       = shr_const_rearth! radius of earth (m)
    real(r8), public           :: tmelt        = shr_const_tkfrz ! Freezing point of water (K)
+   real(r8), public           :: omega        = shr_const_omega ! earth rot ~ rad/sec   
```

Comment:
Okay, I see why you used omega for the namelist parameter now, it refers to the shr_const_omega value. Hmm, still, seems a better name would be useful. I don't feel strongly about this though.

Reply:
if you don't feel strongly, then I would prefer to leave it as omega.